docs/contents/releases.md

```diff
@@ -0,0 +1,90 @@
+# Releases
+
+We use the following different release statuses:
+
+* [PROPOSED](#proposed)
+* [PLANNED](#planned)
+* [BUILD](#build)
+* [DEVELOPER PREVIEW](#developer-preview)
+* [BETA](#beta)
+* [GENERALLY AVAILABLE](#generally-available) (GA)
+
+## PROPOSED
+
+The `PROPOSED` status indicates that **the AWS service is on our radar** for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL NOT** be associated with a GitHub Milestone.
+
+## PLANNED
+
+The `PLANNED` status indicates that the AWS service is on our radar for
+building into ACK.
+
+There will be a Github Issue for tracking the build of the ACK service
+controller for the service.
+
+The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
+target date for the `DEVELOPER PREVIEW` release of that service's controller.
+
+## BUILD
+
+The `BUILD` status indicates that the ACK service controller for the AWS
+service is **actively being built** in preparation for `DEVELOPER PREVIEW`
+release of that ACK service controller.
+
+## DEVELOPER PREVIEW
+
+The `DEVELOPER PREVIEW` status indicates that the **source code** for ACK
+service controller for the AWS service has been check into the ACK source
+repository along with **minimual end-to-end test cases** that run against a
+local Kubernetes-in-Docker (KinD) cluster.
+
+Notably, an ACK service controller in `DEVELOPER PREVIEW` **DOES NOT include
+Helm charts or published binary Docker images** for easy installation of the
+controller.
+
+The following are release criteria for `DEVELOPER PREVIEW`:
+
+* Source code for the controller checked into ACK source repository
+* Mininal (smoke) tests for at least one resource
+* Documentation on to test the controller using KinD
+
+## BETA
+
+The `BETA` status indicates that the ACK service controller for the AWS service
+**has Helm charts and published binary Docker images** that users can use to
+easily install and configure the controller.
+
+In addition, ACK service controllers in `BETA` status have **more extensive
+end-to-end tests** included in the source repository and the successful running
+of these tests **gate any changes to the service controller source code**.
+
+The following are release criteria for `BETA`:
+
+* Comprehensive end-to-end tests for at least one resource
+* Documentation on how to install and configure the controller using Helm
+* Published binary Docker image for the controller
+
+## GENERALLY AVAILABLE
+
+An ACK service controller reaches the `GENERALLY AVAILABLE` (GA) release status
+once a controller in the `BETA` release status has satisfied the following
+criteria:
+
+* Full end-to-end test coverage of all CRUD operations for all resources in the
+  API
+* Test coverage for "negative" or "fuzz" testing to ensure validating webhooks
```

Comment:
We also need soak testing (let's say at least 1 week for starters?)

Reply:
Added